reference/configuration/framework.rst

```diff
@@ -1692,6 +1694,19 @@ method.
     For more information, see
     :ref:`component-serializer-handling-circular-references`.
 
+mapping
```

Comment:
The target `mapping` is duplicated (already used for the `validation` section)

Reply:
Thanks! I've fixed it.